helm-charts/templates/data-plane/gateway-components/gateway-runtime/router-health-probe-script-conf.yaml

```diff
@@ -0,0 +1,44 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+{{ if or .Values.wso2.apk.dp.enabled .Values.wso2.apk.cp.enabled }}
+
+apiVersion: v1
```

Comment:
not requied since dockerfile copying the file

Reply:
removed and resolved